src/plugin-dynamic-import-retry/import-wrapper.ts

```diff
@@ -0,0 +1,72 @@
+/**
+ * This function will be stringified and injected into bundle.
+ * Make sure everything is literally inlined, no imports or external references.
+ */
+export function createImportWrapper(options = { retries: 3 }) {
+  type ImportFn = () => Promise<any>
+
+  const identity = (e: any) => e
+  const retry = createDynamicImportWithRetry(options.retries)
+
+  /*
```

Comment:
Which part of this file is from the external reference? Can we move the copyright and the code into a separate file, add a URL from where the code was sourced from, and import here where needed?

Reply:
The link has been added.

> Can we move the copyright and the code into a separate file

No, the `createImportWrapper` function will be `toString()` and be injected into the source code, any `import` won't work at runtime. I've put explanation comment at the top